packages/react-native-editor/README.md

```diff
@@ -21,13 +21,7 @@ Note that the OS platform used by the maintainers is macOS but the tools and set
 
 * Clone the project and submodules:
```

Comment:
```suggestion
* Clone the project:
```
would it be more proper wording now?

Reply:
Yes! I committed this directly, thanks 👍